FEXHeaderUtils/FEXHeaderUtils/Filesystem.h

```diff
@@ -19,6 +19,12 @@
 #include <unistd.h>
 
 namespace FHU::Filesystem {
+#ifdef _WIN32
```

Comment:
These preprocessor branches add a lot of unnecessary noise that make the interfaces in this file difficult to read.

Instead of adding one such branch per function, could you group them together? (Doesn't need to be all in one block, but it might make sense to reorder some function definitions for readability as well)